Updates SCGF and SGCEC uniforms

SomeAngryMiner opened the pull request:
I can't believe i am writing this at like. almost 3am in the morning.

Before i go into the thick of the changes done, i would like to add a small writer's note:


A Ticket (874-35093) will be opened for staff to discuss changes, opinions or amendments required, if needed. I am _more than willing_ to modify stuff or do extra work, to get this one done for. Can't stress that enough. It was long overdue.
Such a thing happened with a prior PR, #32258 , And from my recollections of such, we could reach an agreement quite nicely.
-

God help you if you're going to read all of the below.

So! on the changes themselves:

**Wait what? what is this?**

For the extremely short story: This resprites EC and Fleet utility uniforms, keeping (most of) the visual designs intact, with some exceptions.

That doesn't explain much, though.

In the longer version? These sprites are old. _really_ old. They use shape patterns (most notably around the backhand and thigh) of human sprites no longer in use. They use colors that are quickly getting outphased by new ones, such as the new navy color set used by ERTs, Armors, Hullbreakers, Drones. The list goes on. 
In short, they are going obsolete, and that's starting to show, and will become more and more pronounced as more sprites come in.

Besides some reasons i will develop for the rest of developers to see on the ticket, i understand how important they are, and how prickly the subject is to some. Which is why i still stress the fact i am more than open to make compromises, to make sure the server's style moves forward.

A few of the changes are not precisely drastic, such as on this image:
![image](https://github.com/user-attachments/assets/c93be021-a88a-4511-8c6c-73e74dff9b2a)
Its still, very much, the EC uniform, but with some adjustments in color and style, it carries less noise for people to strap all their doohickeys and decoration on top without creating a mangled sprite mess, while sticking to a standarized color set. And the rank pins are actually visible.

This encompasses the bulk of the PR, and it affects, as one might have guessed, other uniforms, such as:
![image](https://github.com/user-attachments/assets/aa02a10a-4c25-4948-b319-026c5831ab70)
Fleet fatigues,
![image](https://github.com/user-attachments/assets/20dd162f-356c-4f4e-aeab-f03cb6a62796)
EC service,
![image](https://github.com/user-attachments/assets/2af9b7a9-432c-42ae-a31d-68861bad28db)
Or Fleet Dress shirts,
This change affects Berets. PT uniforms. Covers. Caps. Everything inbetween.

With some exceptions.
A few have recevied some more drastic changes,
![image](https://github.com/user-attachments/assets/8762f75a-0867-4ea6-823a-97d1e0f03cc4)
Namely, the EC dress shirt has been completely wiped and made into a much lighter gray, instead of dark gray to put some chromatic balance on it,
![image](https://github.com/user-attachments/assets/fa94d795-ec87-437a-9d6e-bd32386ace60)
Or the fleet utility cap having their flip-down cover be reduced to ears, instead of whole face, which conflicted with any kind of mask, and was oversized in comparison to even hardhats.

Not pictured: Similar changes struck departmental ushankas (Old ones were literally JPEGs!), among others.

And, finally, others have received ground-up rework, due to the designs being either too simplistic, or weren't particularly well received by the playerbase as a whole because of their quality.

![image](https://github.com/user-attachments/assets/4f966a21-876a-477b-88d6-2dccb6c30a33)

From Left to Right: XPL, CXPL, OFCR and CAPT dress uniforms.
White Uniform Captain is a far shot from the old one, but i believe making a marking distinction between the captain and the rest of the crew, /atleast/ in dress uniform is benefitial from a gameplay perspective, and theatricals for narration and imagery.
Not pictured: CDR still gets their white hat. That one remains simply updated to new style.

![image](https://github.com/user-attachments/assets/0b5b5160-ba9d-4d80-89a2-579e10ac1cc5)
From Left to  Right: Crewman (garrison cap), PO2 (wheel cover), NCSO, OFCR and CDR service uniforms.
Old design did not translate well into the colors, and as such, i would like to offer a modern, more detailed alternative. You're not meant to have that many accessories over these, after all.

![image](https://github.com/user-attachments/assets/36e3b399-6130-4c5b-a445-cf0dfcbffd8b)
In the same vein as the above: Dress uniforms.

This covers about the bulk of the PR. While it is a small change, from a programming perspective, it is still due some severe gravitas because of the centrality of the changes.

Why is still marked as WIP?
-

There are a few things i would like some input on how to handle.
Namely:
- Fleet coveralls.
- Should the White dress for the captain stay?
- How do we handle fleet ranks, considering that they clip over the shoulders of most sprites? would an alternative sprite or just adjusting the existing pattern would be more preferred?
- I would like to go over some general changes before going forward with the rest (eg: Considering they are so similar, it possibly would be preferable to squash the departmental winter coats and ushankas into general 'torch' themed gear, since they're not part of utility uniforms or the like strictly.)
- I would like to make it so these are unbuttonable for RP/visualization reasons. unbutton yourself if you're a mess of an enlisted, drunk, or want to show a concealed vest under it.
- General small tweaks.

Why are some sprites "removed"?
-

A vast majority of those are unused army sprites, which are contained on their own "faction" pack as of #33702 , And were left behind as uncleaned remnants of when the army resided on the torch.
Other examples include an... old service skirt? for EC? it did not even have a path anymore, and i could not find the PR that killed them, and as such, it has been treated under the same nature as the above.
polos /did/ exist. In a very theoretical world. added by #28069 , then amplified by #28221, it was later killed by #30434 from torch equipment, then promptly forgotten by everyone. They do not bring anything to the table that can't be covered by fatigues or coveralls, and as such, i believe they are redundant.

Why in the ever loving fuck is human.dmi touched?
-
I.
I wish i had a better answer.
The /underwear sprites/ are stored on that file. I dont know why that file /exists/, other than storing some dummies, old husk sprites, and fat human sprites, besides the entirety of underwear piled there. Its like a fucked up metaphor of a bunch of trashbags thrown on the back alley of the server.
Once i adapted the new uniforms to use the current body shape, i was horrified by the fact that the spaceman buttocks were so big they ripped space and time everytime they turned to the side, like some kind of 2021 tiktok meme.
So. Uh. Yeah.
Sidestates of underwear had some extremely minimal trimming, quite literally imperceptible, and is otherwise completely unchanged.

If you made it this far. Thank you, and sorry for reading this small novel.

Also a massive thank you to JoeyNoseGay, PurplePineapple, SilentAutumn, 10sc, PapiSmirnov and Gylta23 for providing the feedback and help required for this initial draft to come to life.

🆑 Azlan
rscadd: Resprites EC and Fleet Utility, service and dress uniforms, and all associated accessories.
rscdel: Nerfs spaceman ass.
/:cl:

SomeAngryMiner committed:
Updates SCGF and SGCEC uniforms

SomeAngryMiner commented:
Ammended 8 point cover front and back states. Should be closer to old sprite now.

SierraKomodo reviewed (dismissed):
Rebase

SomeAngryMiner committed:
Merge remote-tracking branch 'upstream/dev' into uniforms

SomeAngryMiner commented:
rrrrebased.

SomeAngryMiner committed:
Winter coats and patch

SomeAngryMiner commented:
Departmental fleet patch on the shoulder has been removed from the insignia, after some feedback.
![image](https://github.com/user-attachments/assets/c51e156c-d85f-419c-a760-c8f40c53b40e)
Wintercoats have been updated.

CakeQ commented:
roll-down overlay for fleet fatigues is missing, same for fleet coveralls
![image](https://github.com/user-attachments/assets/c1b60d67-01b6-4d89-a89e-2ac4df1cb8a0)

SomeAngryMiner committed:
spritefixes + service coat adjustment

SomeAngryMiner commented:
Fixed EC XO Cap.
Adjusted sidestate of wheel covers to have their brim be thinner.
(Temporary?) You no longer flash errors during a rolled down uniform. This was related to accessories wanting to assign their icon as "none", which is now used by the error icon, when rolled down. Until further notice, a "blank" state is created for that. Could probably extend that to every accessory so these are not outliers and the proper tag of ACCESSORY_ROLLED_NONE has no deviations, for parity.
The lower end of fleet service coats has been trimmed, and the silver crests have been removed from general enlisted service coats.

![image](https://github.com/user-attachments/assets/afb842b6-6941-4152-aa5f-4b6c241790af)

CakeQ commented:
Fleet officer's cap not representative of obj sprite. Not sure if unintentional or not. The brim is hard to see on white tiling

![image](https://github.com/user-attachments/assets/351de0a1-43e3-492f-9960-1023bebd7470)
![image](https://github.com/user-attachments/assets/2b17982a-f179-483b-9bb4-261425ec4035)
![image](https://github.com/user-attachments/assets/2090fa1a-8b17-439e-b0ee-9ca3ceaa2223)

SomeAngryMiner committed:
unathi + final adjustments

SomeAngryMiner commented:
Removed the previous "blank" state from the prior commit now that i made sure nothing is erroring out.
Unathi DMIs updated.
Minor errors like EC pin sidestate colors ammended.

That plus some other minor changes, i would call this ready to review.

SierraKomodo commented:
Spookerton wants to do a review pass on this when they have time and so do I. Tagging accordingly.

SierraKomodo reviewed (approved):
Right lets see how this goes.
